crates/bevy_ecs/src/schedule/executor/simple.rs

```diff
@@ -36,73 +35,11 @@ impl SystemExecutor for SimpleExecutor {
         world: &mut World,
         _skip_systems: Option<&FixedBitSet>,
```

Comment:
Nit: since we're actually using this argument, I think that we should remove the _ from the name.